lib/opensusebasetest.pm

```diff
@@ -440,7 +441,8 @@ sub wait_boot {
             workaround_type_encrypted_passphrase;
             assert_screen "grub2", 15;
         }
-        elsif (!match_has_tag("grub2")) {
+        # If KEEP_GRUB_TIMEOUT is set, SUT may be at linux-login already, so no need to abort in that case
+        elsif (!match_has_tag("grub2") and !match_has_tag('linux-login')) {
             # check_screen timeout
             die "needle 'grub2' not found";
```

Comment:
that could be updated now depending on the actual value of 'KEEP_GRUB_TIMEOUT' to not confuse

Reply:
Done: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6384